dataproc/noxfile.py

```diff
@@ -45,6 +45,10 @@ def blacken(session):
     """Run black.
 
     Format code to uniform standard.
+    
+    This currently uses Python 3.6 due to the automated Kokoro run of synthtool.
+    That run uses an image that doesn't have 3.6 installed. Before updating this
+    check the state of the `gcp_ubuntu_config` we use for that Kokoro run.
```

Comment:
Groan.  So we have a bot scream coming, but at least it will be easy to approve them all.